app/src/main/java/com/example/lexis/utilities/Utils.java

```diff
@@ -74,4 +83,38 @@ public static Calendar getYesterday() {
         cal.add(Calendar.DATE, -1);
         return cal;
     }
+
+    /*
+    Return the current target language of the logged-in user.
+    */
+    public static String getCurrentTargetLanguage() {
+        ParseQuery<ParseUser> query = ParseUser.getQuery();
+        query.whereEqualTo("objectId", ParseUser.getCurrentUser().getObjectId());
```

Comment:
Does this field also need to be null checked? `ParseUser.getCurrentUser().getObjectId()`

Reply:
Fixed — I actually also ended up spotting + fixing a bug that could cause the app to crash when getting the target language fails, thank you!  😊